docs/source/usage/cli.rst

```diff
@@ -170,11 +170,15 @@ Image
 
 Image-based provisioning can be accomplished through the use of `Metalsmith`_. It requires the image to be uploaded into OpenStack Glance. Once that's done, a non-admin can run the following:
 
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-|                | **Actions**                                                                                                         |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --ssh-public-key <path-to-key>`` |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
++-------------------+---------------------------------------------------------------------------------------------------------------------+
+|                   | **Actions**                                                                                                         |
++-------------------+---------------------------------------------------------------------------------------------------------------------+
+| Provision Node    | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --candidate <node-uuid> --ssh-public-key <path-to-key>`` |
++-------------------+---------------------------------------------------------------------------------------------------------------------+
+| Remove Node       | ``metalsmith undeploy <node-uuid>``                                                                                 |
++----------- -------+---------------------------------------------------------------------------------------------------------------------+
+| Remove Deployment | ``metalsmith undeploy <deployment-uuid>``                                                                           |
```

Comment:
I've always just used `metalsmith undeploy <node id>`, to be honest - does that work?

Reply:
Yes. it works for sure.